php/cubex/config.yaml

```
@@ -0,0 +1,3 @@
+framework:
+  website: cubex.io
+  version: 4.15.0
```

We prefer this format

```suggestion
  version: 4.15
```